FatmaQunnies/utils.py

```diff
@@ -0,0 +1,36 @@
+import re
+from task_status import Status
+
+def validate_date(date_str):
+    while True:
+        if re.match(r'^\d{4}-\d{2}-\d{2}$', date_str):
+            return date_str
+        else:
+            print("Invalid date format. Use YYYY-MM-DD. Please try again.")
+
+def get_valid_input(inputt):
+  while True:
+        
+        if inputt:
+            return inputt
+        else:
+            print("Input cannot be empty. Please try again.")
+ 
+
+def validate_status(status):
+ valid_statuses = ['inProgress', 'completed']
+ while True:
+       
+        if status.upper() in Status.__members__:
```

Comment:
Using the upper() method will convert all the letters to uppercase. `inProgress` != `INPROGRESS`